Components/BlockImageText/functions.php

```diff
@@ -23,10 +23,11 @@
                 'name' => 'image',
                 'type' => 'image',
                 'preview_size' => 'medium',
-                'instructions' => '',
+                'instructions' => 'Minimum width of the image should be 1920px',
```

Comment:
Why? I think this might _break_ it for many users and cases and would prefer no min size. If it's uploaded in smaller size, it should get scaled up in poor quality, but still work.